AudioSinkExample/RecordingsViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  RecordingsViewController.swift
+//  AudioSinkExample
+//
+//  Copyright © 2017 Twilio Inc. All rights reserved.
+//
+
+import Foundation
+import AVKit
+
+class RecordingsViewController: UITableViewController {
+
+    let kReuseIdentifier = "RecordingsCellId"
+    var recordings = Array<URL>()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.title = "Recordings"
+        tableView.register(UITableViewCell.classForCoder(), forCellReuseIdentifier: kReuseIdentifier)
+
+        let fileManager = FileManager.default
+        guard let documentsDirectory = fileManager.urls(for: .documentDirectory, in: .userDomainMask).last else {
+            return
+        }
+
+        var directoryContents : [String]?
+        do {
+            try directoryContents = fileManager.contentsOfDirectory(atPath: documentsDirectory.path)
+        } catch {
+            print("Couldn't fetch directory contents. \(error)")
+            return
+        }
+
+        for path in directoryContents! {
+            if (path.lowercased().hasSuffix("wav")) {
+                recordings.append(URL(fileURLWithPath: path, relativeTo: documentsDirectory))
+            }
+        }
```

Comment:
we can avoid unwrapping by - 

```
        do {
            let directoryContents = try fileManager.contentsOfDirectory(atPath: documentsDirectory.path)
            
            for path in directoryContents {
                if (path.lowercased().hasSuffix("wav")) {
                    recordings.append(URL(fileURLWithPath: path, relativeTo: documentsDirectory))
                }
            }
        } catch {
            print("Couldn't fetch directory contents. \(error)")
            return
        }
```

Reply:
I'd prefer not to force unwrap. Will update.